### PR TITLE
doc/dev/crimson: add instruction on running install-deps.sh for build…

### PR DESCRIPTION
**Pull request by tchaikov**
…ing crimson

Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
@neha-ojha hopefully i can include more stuff on crimson before merging this PR.

**Review by neha-ojha — APPROVED** [review]

**Comment by neha-ojha** [comment]
> @neha-ojha hopefully i can include more stuff on crimson before merging this PR.

sure

**Comment by tchaikov** [comment]
@neha-ojha a copy of the tutorial you mentioned is browsable at one of Seastar's author's home page. and i am trying to fix it at https://groups.google.com/g/seastar-dev/c/8ymBVI9iBas/m/ftVtE50TBAAJ

**Comment by neha-ojha** [comment]
jenkins render docs

**Review by rzarzynski — APPROVED** [review]

**Comment by tchaikov** [comment]
changelog

- pickup the latest seastar
- add instructions on how to build seastar document.

File: doc/dev/crimson.rst [hunk 1]
```diff
@@ -179,6 +179,22 @@ corresponding test results are highlighted.
...
+
+See `Seastar Tutorial <https://github.com/scylladb/seastar/blob/master/doc/tutorial.md>`_ .
+Or build a browsable version and start an HTTP server::
+
+  $ cd seastar
+  $ ./configure.py --mode debug
+  $ ninja -C build/debug docs
+  $ python3 -m http.server -d build/debug/doc/html
+
+You might want to install pandoc and other dependencies beforehand.
```
Comment (rzarzynski) [inline doc/dev/crimson.rst:197]:
```suggestion
You might want to install ``pandoc`` and other dependencies beforehand.
```

**Review by rzarzynski — APPROVED** [review]

File: doc/dev/crimson.rst (hunk 1 above)
Comment (tchaikov) [reply doc/dev/crimson.rst:197]: fixed and repushed.

**Review by rzarzynski — APPROVED** [review]

**Comment by tchaikov** [comment]
jenkins test crimson perf

**Comment by tchaikov** [comment]
jenkins test crimson perf

**Comment by tchaikov** [comment]
jenkins test crimson perf

**Pull request merged**